packages/server/src/unstable-core-do-not-import/stream/utils/asyncIterable.ts

```diff
@@ -81,8 +77,6 @@ export async function* takeWithGrace<T>(
       yield result.value;
       if (--count === 0) {
         timerPromise = timer.start();
-        // eslint-disable-next-line @typescript-eslint/no-floating-promises
-        timerPromise.then(() => opts.abortCtrl.abort());
```

Comment:
Shouldn't be needed as ending the stream will cause the abort controller to be triggered